systemtest/src/test/java/io/strimzi/systemtest/mirrormaker/MirrorMaker2ST.java

```diff
@@ -632,8 +632,8 @@ void testIdentityReplicationPolicy() {
         );
 
         resourceManager.createResourceWithWait(
-            KafkaTemplates.kafkaEphemeral(testStorage.getSourceClusterName(), 1, 1).build(),
-            KafkaTemplates.kafkaEphemeral(testStorage.getTargetClusterName(), 1, 1).build(),
+            KafkaTemplates.kafkaPersistent(testStorage.getSourceClusterName(), 1, 1).build(),
+            KafkaTemplates.kafkaPersistent(testStorage.getTargetClusterName(), 1, 1).build(),
```

Comment:
If you are changing the storage from ephemeral to persistent here, should you update the corresponding NodePools as well? (I guess this applies to most of the places)

Reply:
sure after the discussion this can be done in single PR.

Reply:
thanks for bringing this up  👍